props/saber_BC_buttons.h

```diff
@@ -477,738 +1495,1272 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
         }
       }
     }
-    // EVENT_THRUST
-      if (mss.y * mss.y + mss.z * mss.z < 16.0 &&
-          mss.x > 14  &&
-          fusor.swing_speed() < 150) {
-        if (millis() - thrust_begin_millis_ > 15) {
-          Event(BUTTON_NONE, EVENT_THRUST);
-          thrust_begin_millis_ = millis();
-        }
-      } else {
+// EVENT_THRUST
+    if (mss.y * mss.y + mss.z * mss.z < 20.0 &&  // slightly more forgiving than 16.0?
+       (mss.x > 14 || mss.x < -14) &&  // Check for both positive and negative x-motion
```

Comment:
mss.x > 14 || mss.x < -14   ->  fabs(mss.x) > 14

Reply:
Yes. Done.

Reply:
Doesn't look done?

Reply:
it is now.